src/Microsoft.AspNet.Mvc.Razor/TagHelperContentWrapperTextWriter.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.Razor
+{
+    /// <summary>
+    /// <see cref="TextWriter"/> implementation which writes to a <see cref="TagHelperContent"/> instance.
+    /// </summary>
+    public class TagHelperContentWrapperTextWriter : TextWriter
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TagHelperContentWrapperTextWriter"/> class.
+        /// </summary>
+        /// <param name="encoding">The <see cref="Encoding"/> in which output is written.</param>
```

Comment:
It's not though :/, the encoding has no effect on what gets written to the `TagHelperContent`. Now that this is consolidated, is the encoding necessary? It's a requirement of the abstract `TextWriter` class but does not come into play for what we're using `TagHelperContentWrapperTextWriter` for.